vita3k/renderer/src/gl/texture.cpp

```diff
@@ -106,6 +108,12 @@ static size_t decompress_compressed_swizz_texture(SceGxmTextureBaseFormat fmt, v
         renderer::texture::decompress_bc_swizz_image(width, height, reinterpret_cast<const std::uint8_t *>(data),
             reinterpret_cast<std::uint32_t *>(dest), ubc_type);
         return (((width + 3) / 4) * ((height + 3) / 4) * ((ubc_type > 1) ? 16 : 8));
+    } else if ((fmt >= SCE_GXM_TEXTURE_BASE_FORMAT_PVRT2BPP) && (fmt <= SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII4BPP)) {
+        // TODO, is not perfect for PVRT-II.
+        pvr::PVRTDecompressPVRTC(data, is_2pp, width, height, (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII2BPP) || (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII4BPP),
+            reinterpret_cast<uint8_t *>(dest));
+        // TODO, calcule return maybe not right.
```

Comment:
lgtm, why do you think so?

Reply:
i have tested few value and get same for all so not sur ^^